extensions/binding/org.eclipse.smarthome.binding.hue/src/main/java/org/eclipse/smarthome/binding/hue/handler/HueBridgeHandler.java

```diff
@@ -434,6 +432,32 @@ public void startSearch(List<String> serialNumbers) {
             }
         }
     }
+    
+    /**
+     * Iterate through lightStatusListeners and notify them about a changed ot added light state.
+     * @param fullLight
```

Comment:
Aggreed. (Wow, what hard reviews!)

Will correct this on Monday. 

Von meinem Toaster gesendet

> Am 18.03.2016 um 21:51 schrieb Markus Rathgeb <notifications@github.com>:
> 
> In extensions/binding/org.eclipse.smarthome.binding.hue/src/main/java/org/eclipse/smarthome/binding/hue/handler/HueBridgeHandler.java:
> 
> > @@ -434,6 +432,32 @@ public void startSearch(List<String> serialNumbers) {
> >              }
> >          }
> >      }
> > +    
> > +    /**
> > +     * Iterate through lightStatusListeners and notify them about a changed ot added light state.
> > +     * @param fullLight
> Add empty line before (see comment above)
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
>